src/components/product/ProductCard/product-card.scss

```diff
@@ -84,15 +105,107 @@
     }
   }
 
-  [data-card-actions] {
-    position: relative;
-    margin-top: auto;
-    padding-top: var(--fs-spacing-4);
+  [data-store-badge] { margin-top: var(--fs-spacing-2); }
+}
+
+[data-fs-product-card-title] {
+  color: var(--fs-product-card-text-title-color);
+  font-weight: var(--fs-product-card-text-title-weight);
+  font-size: var(--fs-product-card-text-title-size);
+  line-height: 1.25;
+}
+
+[data-fs-product-card-prices] {
+  display: inline-flex;
+  align-content: center;
+
+  [data-variant="spot"] {
+    color: var(--fs-product-card-text-price-color);
+    font-size: var(--fs-product-card-text-price-size);
+  }
+
+  [data-variant="listing"] {
+    color: var(--fs-product-card-text-price-listing-color);
+    font-size: var(--fs-product-card-text-price-listing-size);
+  }
+}
+
+[data-fs-product-card-footer] {
+  display: flex;
+  gap: var(--fs-product-card-gap);
+  align-items: center;
+  justify-content: space-between;
+  margin-top: var(--fs-product-card-gap);
+
+  small {
+    display: block;
+    color: var(--fs-product-card-text-subtitle-color);
+    font-size: var(--fs-product-card-text-subtitle-size);
+  }
+}
+
+[data-fs-product-card-bordered=true] {
+  border: var(--fs-border-width-default) solid var(--fs-border-color-light);
+}
+
+// --------------------------------------------------------
+// Template Variants
+// --------------------------------------------------------
+
+[data-fs-product-card-variant="wide"] {
+  --fs-product-card-padding: 0;
+  --fs-product-card-content-padding: var(--fs-spacing-2);
```

Comment:
```suggestion
  --fs-product-card-content-padding: var(--fs-spacing-3);
```

|Before|
|-|
![before-space-3](https://user-images.githubusercontent.com/3356699/160215355-5a1fe376-026f-48e5-a8d9-92bd3ac404dd.png)

|After|
|-|
![before-space-2](https://user-images.githubusercontent.com/3356699/160215236-dc1946e4-ecb1-4df0-ac77-ded50bd9e150.png)

Not sure. Should we keep it or change it?

Reply:
`var(--fs-spacing-2)` is the correct one.